features/mirror_spec.rb

```diff
@@ -0,0 +1,23 @@
+require File.expand_path('../support/command_rspec_helper', __FILE__)
```

Comment:
Not actually used in this file -- see my previous comment about using `%x` and `command`.